### DISCUSSION TITLE
Please provide updated example to use HMC in Blackjax

### DISCUSSION THREAD

**Discussion by grahnder** [blackjax-devs/blackjax#546] [General]
Can someone please provide a demo notebook to explain how the api can be used to perform HMC on a probabilistic model. All the example notebooks have outdated.

**Comment by junpenglao** [comment]
We are in the process of updating the documentations, created an issue for tracking: https://github.com/blackjax-devs/blackjax/issues/551

**Reply by junpenglao** [reply]
Actually, the notebooks are tested against `main`: 
![image](https://github.com/blackjax-devs/blackjax/assets/12952641/69289230-7cd1-4d76-b1d3-c3242747dc96)

If you install from `main` or nightly release the example should be working.
